qemu/tests/qemu_guest_agent.py

```diff
@@ -1277,6 +1277,45 @@ def check_value_frontend_open(out, expected):
         check_value_frontend_open(out, False)
         session.close()
 
+    @error_context.context_aware
+    def gagent_check_frozen_io(self, test, params, env):
+        """
+        fsfreeze test during disk io.
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment
+        """
+        error_context.context("Before freeze/thaw the FS, run the iozone test",
+                              logging.info)
+        session = self._get_session(self.params, None)
+        self.session = session
+        self._open_session_list.append(session)
+        iozone_cmd = utils_misc.set_winutils_letter(self.session,
+                                                    self.params["iozone_cmd"])
+        self.iozone_thread = utils_misc.InterruptedThread(self.session.cmd,
+                                                          args=(iozone_cmd, 360))
```

Comment:
@xiagao As iozone create 1G size file.  It will be completed very quickly.   Do it need 360 seconds?